src/components/scenes/newAccount/NewAccountReviewScene.tsx

```diff
@@ -99,10 +99,7 @@ export const NewAccountReviewScene = (props: NewAccountReviewProps) => {
     }
     onLogEvent(`Signup_Review_Done`)
     onLogin(route.params.account)
-
-    dispatch(checkAndRequestNotifications(branding)).catch(error =>
-      console.log(error)
-    )
+    dispatch(completeLogin(route.params.account))
```

Comment:
The `completeLogin` function also calls `onLogin`, so the line above is redundant.